modules/ROOT/pages/platform/api/overview.adoc

```diff
@@ -48,6 +49,19 @@ In the case of `429 Too Many Requests`, we would recommend slowing down the rate
 
 For other `4xx` client error responses, you should not resend such requests without first correcting them.
 
+[NOTE]
+====
+We currently set a global rate limit of 125 requests per minute per client ID for all Aura API endpoints.
+Starting 27 Jan 2025, we will adjust the rate limiting for the `/oauth/token` endpoint, which is used to obtain authorization tokens. 
+The new limit will be set to 15 requests per hour.
+
+Since each authorization token remains valid for one hour, we recommend optimizing your API usage in advance by following these steps:
+
+* *Retrieve the token once per hour:* As tokens remain valid for an hour, reduce the frequency of token requests and reuse the same token for multiple API calls.
+
+* *Implement token caching:* Store the token securely within your system to reuse it for subsequent requests during its validity period, minimizing unnecessary calls to the endpoint.
```

Comment:
```suggestion
Currently the global rate limit is set to 125 requests per minute per client ID for all Aura API endpoints.
Shortly, the rate limiting for the `/oauth/token` endpoint, which is used to obtain authorization tokens will be adjusted. 
The new limit will be set to 15 requests per hour.

Since each authorization token remains valid for one hour, it is recommended that you optimize your API usage in advance by following these steps:

* *Retrieve the token once per hour:* As tokens remain valid for an hour, reduce the frequency of token requests and reuse the same token for multiple API calls.

* *Implement token caching:* Store the token securely within your system to reuse it for subsequent requests during its validity period, minimizing unnecessary calls to the endpoint.
```